src/PoseNet/index.js

```diff
@@ -74,15 +74,26 @@ class PoseNet extends EventEmitter {
     }
 
     const pose = await this.net.estimateSinglePose(input, this.imageScaleFactor, this.flipHorizontal, this.outputStride);
+
     const result = [{ pose, skeleton: this.skeleton(pose.keypoints) }];
     this.emit('pose', result);
+
     if (this.video) {
       return tf.nextFrame().then(() => this.singlePose());
     }
+
+    if (typeof cb === 'function') {
+      if (typeof result === 'object') {
+        cb(null, result);
+      } else {
+        cb(new Error('result is not valid'));
```

Comment:
`result` is always an array, no? Why check for `object`

Reply:
Ah you're right. In terms of error handling, maybe then checking if an array is not returned would be better. e.g.:

```js
      if (typeof result !== 'object') {
        cb(new Error('expected array or object, got something else'));
      } else {
        cb(null, result);
   }
```

Reply:
Perhaps this error check is not necessary. Since this happens before:
```javascript
const poses = await this.net.estimateMultiplePoses(input, this.imageScaleFactor, this.flipHorizontal, this.outputStride);
const result = poses.map(pose => ({ pose, skeleton: this.skeleton(pose.keypoints) }));
``` 
it will throw an error if `poses` is not an array and `poses.map` will not run.

Reply:
@cvalenzuela - thanks for the feedback. Yes, you're right the error check may be extra for now. I will remove it and if we find a case in which error cases pop up, we can add in later. For now, let's just return the results. 

I will update the PR. Thanks!